server/api_router.go

```diff
@@ -98,6 +98,7 @@ func (p *Plugin) newAPIRouter() *mux.Router {
 	hostCtrlRouter.HandleFunc("/make", p.handleMakeHost).Methods("POST")
 	hostCtrlRouter.HandleFunc("/mute", p.handleMuteSession).Methods("POST")
 	hostCtrlRouter.HandleFunc("/screen-off", p.handleScreenOff).Methods("POST")
+	hostCtrlRouter.HandleFunc("/unraise-hand", p.handleUnraiseHand).Methods("POST")
```

Comment:
I'd also expect this to become `lower-hand`, and in the new methods as well.

Reply:
🤦